charts/ui/values.yaml

```diff
@@ -18,6 +18,7 @@ ui:
   recaptchaSitekeySecretKey: ""
   apiUrl: ""
   subdomainSuffix: ""
+  cnameRecord: ""
```

Comment:
It's not essential but probably we want to make sure the the image tag is bumped from `3.2` to whatever we end up releasing from the ui repo so that the image is capable of using this value.

Otherwise we'll have to release this chart twice

Reply:
set it to imaginary 3.3 now will add a changelog row to https://github.com/wbstack/ui/pull/336